pkg/arvo/gen/ph/cancel.hoon

```diff
@@ -1,3 +1,4 @@
+::  Cancel a ph command
```

Comment:
The entire gen/ph directory should be deleted now that app/ph no longer exists (it's a library now and doesn't use generators)